README.md

```diff
@@ -1,302 +1,293 @@
 
-    ```js
-    document.body.dispatchEvent(new CustomEvent('oTracking.page', { detail: { content: { uuid: 'abc-123', barrier: 'PREMIUM' }}, bubbles: true}));
-    ```
-- `oTracking.event`
+```js
+const opts = {
+    category: 'audio', // default: 'component'
+    selector: '.o-teaser__audio', // default: '[data-o-tracking-view]'
+    getContextData: (el) => {  // default: null
+        return {
+            componentContentId: el.getAttribute('data-id'),
+            type: 'audio',
+            subtype: 'podcast',
+            component: 'teaser'
+        };
+    }
+};
 
-    Send a normal event
+oTracking.view.init(opts);
+```
 
-    ```js
-    document.body.dispatchEvent(new CustomEvent('oTracking.event', { detail: { category: 'video', action: 'play', id: '512346789', pos: '10' }, bubbles: true}));
-    ```
+##### oTracking.event
 
-## Parameters
+To track a custom event call the `oTracking.event` method. For example to track that a video is played:
+```js
+const eventConfig = {
+    "category": "video", // optional
+    "action": "play", // optional
+};
+oTracking.event(eventConfig);
+```
+Please refer to the [event document](docs/event.md) for information about all the possible properties which can be set.
 
-o-tracking will
-* Automatically pickup ftsession from cookies for you.
-* Page events automatically pick up the url and the referrer.
-* Click events [initalised as above](#usage), will populate a `domPathTokens` property. If the clicked element has the `data-trackable` attribute, sibling elements will also be included within `domPathTokens`.
-* View events are fired for elements with the `data-o-tracking-view` attribute by default, unless `o-tracking`'s `selector` option is configured. Like click events, view events populate a `domPathTokens` property. To collect data for events, set the `category` option, or provide a callback[`getContextData`]
-_Note:_ This feature requires `window.IntersectionObserver` in order to track the events
-_Note:_ `getContextData` should be a function which returns `{Object}`. It accepts the viewed element as an argument
-	```js
-	const opts = {
-		category: 'audio', // default: 'component'
-		selector: '.o-teaser__audio', // default: '[data-o-tracking-view]'
-		getContextData: (el) => {  // default: null
-			return {
-				componentContentId: el.getAttribute('data-id'),
-				type: 'audio',
-				subtype: 'podcast',
-			 	component: 'teaser'
-			};
-		}
-	}
 
-	oTracking.view.init(opts);
-	```
+#### Events
 
-Events are decorated with config values you pass in via `init()` or `updateConfig()` if they are part of `context`, `user`, or `device` objects. Values passed in as `context` to individual events will override context values from init.
+Instead of calling the `page` and `event` methods `o-tracking` can be configured to fire events based on the custom DOM Events `oTracking.page` and `oTracking.event`.
```

Comment:
it would be good to expand here: why would I choose one approach over the other? Would it make sense to do both?

Reply:
It's something I've only seen the FT App use but I can't see anything which fires the 'oTracking.page' event -- https://github.com/Financial-Times/ft-app/blob/83e4c16b34f79fe2923c9361f2e315cb2bc80695/lib/javascript/tracking/events.js#L11-L20